testdata/p4_16_samples/issue363-bmv2.p4

```diff
@@ -0,0 +1,88 @@
+#include <v1model.p4>
+
+struct alt_t {
+  bit<1> valid;
+  bit<7> port;
+};
+
+struct row_t {
+  alt_t alt0;
+  alt_t alt1;
+};
+
+struct local_metadata_t {
+  row_t row;
+};
+
+header bitvec_hdr {
+  row_t row;
+}
+
+struct parsed_packet_t {
+    bitvec_hdr bvh;
+    bitvec_hdr bvh1;    
+};
+
+struct tst_t {
+    bitvec_hdr bvh0;
+}
+
+parser parse(packet_in pk, out parsed_packet_t h,
+             inout local_metadata_t local_metadata,
+             inout standard_metadata_t standard_metadata) {
+  state start {
+//      pk.extract(h.bvh);
+//    pk.extract(h.bvh1);
+    transition accept;
+  }
+}
+
+control ingress(inout parsed_packet_t h,
+                inout local_metadata_t local_metadata,
+	        inout standard_metadata_t standard_metadata) {
+    apply {
+        tst_t s;
+
+        // bit<48> mac;
```

Comment:
I don't understand what you are testing here. The interesting code seems to be commented-out.

Reply:
This code will be removed.  It was to test Cast, but this PR is not going to support Cast.